integration/tests/cook/test_basic.py

```diff
@@ -2025,9 +2025,11 @@ def test_instance_stats_success(self):
         job_uuid_3, resp = util.submit_job(self.cook_url, command='sleep 2', name=name, cpus=0.4, mem=128)
         self.assertEqual(resp.status_code, 201, msg=resp.content)
         job_uuids = [job_uuid_1, job_uuid_2, job_uuid_3]
+        util.wait_for_jobs(self.cook_url, job_uuids, 'completed')
+        instances = [util.wait_for_instance(self.cook_url, j) for j in job_uuids]
```

Comment:
Since these two lines were moved out of the `try` block, we no longer kill the jobs if they fail. Right?

Maybe we should add a second try/catch-raise layer for the sandbox dumping, since dumping the sandbox only when the test failed might be an improvement as well.

Reply:
Good call, done